app/models/ingest_measurement.rb

```diff
@@ -11,5 +11,117 @@ class << self
     def data_for(query)
       search(query: query).results.map(&:_source)
     end
+
+    def query_sensor_data(device_urn) # rubocop:disable Metrics/MethodLength
+      search "query":
+                 { "bool": {
+                   "filter": [
+                     {
+                       "range": {
+                         "@timestamp": {
+                           "gte": 'now-8w',
+                           "lte": 'now',
+                           "format": 'epoch_millis'
+                         }
+                       }
+                     },
+                     {
+                       "query_string": {
+                         "analyze_wildcard": true,
+                         "query": "device_urn:" + "\"" + device_urn + "\"" # rubocop:disable all
```

Comment:
You can make this cleaner with a [term query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-term-query.html)

It should just be
```ruby
{
  "term": {
    "device_urn": device_urn
  }
}
```